Description: Move test and friends to /bin.
Author: Colin Watson <cjwatson@ubuntu.com>,
        Steve Langasek <steve.langasek@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/145382
Forwarded: not-needed
Last-Updated: 2017-10-25

Index: busybox-1.27.2-1ubuntu1/coreutils/test.c
===================================================================
--- busybox-1.27.2-1ubuntu1.orig/coreutils/test.c
+++ busybox-1.27.2-1ubuntu1/coreutils/test.c
@@ -46,9 +46,9 @@
 //config:	help
 //config:	  Enable 64-bit support in test.
 
-//applet:IF_TEST(APPLET_NOFORK(test, test, BB_DIR_USR_BIN, BB_SUID_DROP, test))
-//applet:IF_TEST1(APPLET_NOFORK([,   test, BB_DIR_USR_BIN, BB_SUID_DROP, test))
-//applet:IF_TEST2(APPLET_NOFORK([[,  test, BB_DIR_USR_BIN, BB_SUID_DROP, test))
+//applet:IF_TEST(APPLET_NOFORK(test, test, BB_DIR_BIN, BB_SUID_DROP, test))
+//applet:IF_TEST1(APPLET_NOFORK([,   test, BB_DIR_BIN, BB_SUID_DROP, test))
+//applet:IF_TEST2(APPLET_NOFORK([[,  test, BB_DIR_BIN, BB_SUID_DROP, test))
 
 //kbuild:lib-$(CONFIG_TEST)  += test.o test_ptr_hack.o
 //kbuild:lib-$(CONFIG_TEST1) += test.o test_ptr_hack.o
